src/Symfony/Component/Filesystem/Filesystem.php

```diff
@@ -358,34 +358,37 @@ public function makePathRelative($endPath, $startPath)
             $startPath = str_replace('\\', '/', $startPath);
         }
 
+        $stripDriveLetter = function ($path) {
+            if (strlen($path) > 2 && substr($path, 1, 2) === ':/' && ctype_alpha($path[0])) {
```

Comment:
instead of `substr()` I suggest to use `':' === $path[1] && '/' === $path[2]`

Reply:
👍  for @xabbuh's suggestion

Reply:
Done in bce33c4925be1eeea6b9c56a1e934d874c67f841